exoscale_auth.py

```diff
@@ -3,9 +3,9 @@
 import hashlib
 import hmac
 import time
-
 from base64 import standard_b64encode
-from urllib.parse import urlparse, parse_qs
+from datetime import datetime
```

Comment:
AFAIK the `datetime` import is not used and can be removed

Reply:
It is in `ExoscaleAuth` (not v2) and was incorrectly removed.